_includes/components/charts/chart.html

```diff
@@ -7,16 +7,13 @@
     {% assign graph_template = 'components/charts/' | append: include.graph_type | append: '.html' %}
     {% assign graph_title = page.indicator.graph_titles.first.title | default: page.indicator.graph_title %}
     <div id="selectionsChart">
-      <figure role="group">
```

Comment:
@brockfanning sorry, I'm wondering we should just remove `figcaption` but still keep it wrapped in `figure`?

Reply:
Gotcha, just added it back.